src/Mgate/SuiviBundle/Resources/views/Etude/Buttons/Intervenant.html.twig

```diff
@@ -5,10 +5,10 @@
  %}
  #}
 {% if mission.intervenant %}
-    <div class="btn-group" style="width: 220px;">
+    <div class="btn-group" style="width: 100%; min-width: 220px; max-width:500px;">
         <a class="btn btn-primary"
            href="{{ path('MgatePersonne_membre_voir', {'id': mission.intervenant.id|default('0')}) }}"
-           style="width : 60%; text-align: left; overflow: hidden;">
+           style="width : calc(100% - 26px); text-align: left; overflow: hidden;">
```

Comment:
Need to have a shorter first button of 26px to give place at the dropdown button who is 26px large.